partner_disable_gravatar/models/partner.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2018-2019 Akretion France (https://akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    def _get_gravatar_image(self, email):
+        return False
```

Comment:
You could put an ```# pragma: no cover``` at the end of this (trivial) line to prevent the checks from failing because of lowered coverage.